src/webui/www/private/speedlimit.html

```diff
@@ -19,140 +19,144 @@
-                    break;
-            }
-        });
-
-        const params = new URLSearchParams(window.location.search);
-        const type = params.get("type");
-        const hashes = params.get("hashes").split("|");
-
-        const isGlobal = (hashes[0] === "global");
-        // Otherwise it's download limit
-        const isUpload = (type === "upload");
-
-        const getLimitMethod = isUpload ? "uploadLimit" : "downloadLimit";
-        const setLimitMethod = isUpload ? "setUploadLimit" : "setDownloadLimit";
-
-        const limitUpdateValue = document.getElementById("limitUpdateValue");
-        const limitUnit = document.getElementById("limitUnit");
-
-        const setLimitUpdateValue = (value) => {
-            if (value === 0) {
-                limitUpdateValue.value = "∞";
-                limitUnit.style.visibility = "hidden";
-            }
-            else {
-                limitUpdateValue.value = value;
-                limitUnit.style.visibility = "visible";
-            }
-        };
-
-        const setupSlider = (limit, maximum) => {
-            const input = document.getElementById("limitSliderInput");
-            input.max = maximum;
-            input.value = limit;
-            input.addEventListener("input", (event) => {
-                setLimitUpdateValue(Number(event.target.value));
-            });
-            // Set default value
-            setLimitUpdateValue(limit);
-        };
-
-        const saveLimit = () => {
-            const limit = Number(limitUpdateValue.value) * 1024;
-            if (isGlobal) {
-                fetch(`api/v2/transfer/${setLimitMethod}`, {
-                        method: "POST",
-                        body: new URLSearchParams({
-                            limit: limit
-                        })
-                    })
-                    .then((response) => {
-                        if (!response.ok)
-                            return;
-
-                        window.parent.updateMainData();
-                        window.parent.qBittorrent.Client.closeFrameWindow(window);
-                    });
-            }
-            else {
-                fetch(`api/v2/torrents/${setLimitMethod}`, {
-                        method: "POST",
-                        body: new URLSearchParams({
-                            hashes: hashes.join("|"),
-                            limit: limit
-                        })
-                    })
-                    .then((response) => {
-                        if (!response.ok)
-                            return;
-
+        window.addEventListener("DOMContentLoaded", (event) => {
+            const applyButton = document.getElementById("applyButton");
+
+            window.addEventListener("keydown", (event) => {
+                switch (event.key) {
+                    case "Enter":
+                        event.preventDefault();
+                        applyButton.click();
```

Comment:
Please move unrelated changes to their own commit.

Reply:
It is required for the function continue to work. Otherwise it will be a broken change.